apps/judicial-system/backend/src/app/modules/indictment-count/dto/updateIndictmentCount.dto.ts

```diff
@@ -22,4 +23,8 @@ export class UpdateIndictmentCountDto {
   @IsString()
   @ApiPropertyOptional()
   readonly legalArguments?: string
+
+  @IsOptional()
+  @ApiPropertyOptional()
```

Comment:
```suggestion
  @ApiPropertyOptional({enum: IndictmentCountOffense, isArray: true})
```

☝️ This will help Swagger present a nicer UI.